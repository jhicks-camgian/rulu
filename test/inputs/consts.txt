(TestFact (str "a") (num 1))
(TestFact (str "b") (num 2))
