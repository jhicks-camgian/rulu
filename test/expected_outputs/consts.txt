(OutputStr (str "b"))
(OutputNum (num 1))
